modules/openapi-generator/src/test/resources/3_0/rust-server/ops-v3.yaml

```diff
@@ -0,0 +1,192 @@
+# Test large number of configured operations
```

Comment:
😻 the new test!

Would be nice if this had a more descriptive name than `ops-v3`, but not essential.